lib/astronoby/coordinates/ecliptic.rb

```diff
@@ -17,9 +17,9 @@ def initialize(latitude:, longitude:)
       #  Chapter: 4 - Orbits and Coordinate Systems
       def to_equatorial(epoch:)
         mean_obliquity = Astronoby::MeanObliquity.for_epoch(epoch)
-        obliquity_in_radians = mean_obliquity.value.to_radians.value
-        longitude_in_radians = @longitude.to_radians.value
-        latitude_in_radians = @latitude.to_radians.value
+        obliquity_in_radians = mean_obliquity.value.radians
+        longitude_in_radians = @longitude.radians
+        latitude_in_radians = @latitude.radians
```

Comment:
If you implement math operators on `Angle`, I think a lot of the complexity here goes away. You would no longer need to care that these angles are expressed in radians. All you care is that you have a series of `Angle` objects and you can do math on them.